Marlin/ultralcd.cpp

```diff
@@ -914,125 +912,165 @@ void lcd_cooldown() {
       line_to_current(Z_AXIS);
     #endif
     st_synchronize();
-    mbl_wait_for_move = false;
+  }
+
+  static void _lcd_level_goto_next_point();
+
+  static void _lcd_level_bed_done() {
+    if (lcdDrawUpdate) lcd_implementation_drawedit(PSTR(MSG_LEVEL_BED_DONE));
+    lcdDrawUpdate =
+      #if ENABLED(DOGLCD)
+        LCDVIEW_CALL_REDRAW_NEXT
+      #else
+        LCDVIEW_CALL_NO_REDRAW
+      #endif
```

Comment:
This is the kind of change I did to make sure graphical displays get a complete update. But it's a workaround for the "bad idea" it started with. The value of `lcdDrawUpdate` shouldn't be changed _between graphical draw calls_. So it probably works to simply not change `lcdDrawUpdate` here at all!